tests/test_fio_workload.py

```diff
@@ -16,12 +17,38 @@
 logger = logging.getLogger(__name__)
 
 
+@pytest.fixture(scope='class')
+def test_fixture(request):
+    """
+    Fixture only for teardown
+    """
+
+    self = request.node.cls
+
+    def finalizer():
+        teardown(self)
+    request.addfinalizer(finalizer)
+
+
+def teardown(self):
+    logger.info(f"Deleting pod {self.pod_obj.name}")
+    self.pod_obj.delete()
```

Comment:
If a pod creation will fail during the test, delete here will fail and teardown will be aborted.
The same applies for the below pvc and pv delettions

Reply:
Done, added a check whether pod or pvc exists.